pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -717,6 +717,28 @@ def test_agg_relabel_multiindex_duplicates():
     tm.assert_frame_equal(result, expected)
 
 
+def test_groupby_aggregate_empty_key():
+    # GH: 32580
+    df = pd.DataFrame({"a": [1, 1, 2], "b": [1, 2, 3], "c": [1, 2, 4]})
```

Comment:
parameterize over the 2 cases here (result, result_2) (you can put the kwargs in the parameterization)

Reply:
Thanks! Done.